examples/online-boutique/invalid.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
can you change that file content with just `@` to clearly show it's invalid?

Reply:
Done